addons/point_of_sale/tools/posbox/overwrite_before_init/etc/init_posbox_image.sh

```diff
@@ -49,13 +54,13 @@ password="$(openssl rand -base64 12)"
 echo "pi:${password}" | chpasswd
 
 PKGS_TO_INSTALL="
+    chromium-browser \
     console-data \
     cups \
     cups-ipp-utils \
     dbus \
     dbus-x11 \
     dnsmasq \
-    firefox-esr \
```

Comment:
firefox needs to be kept as when connecting the new image to a database in v16 ==> "no firefox" error